tests/nemo_text_processing/en/data_text_normalization/test_cases_date.txt

```diff
@@ -19,7 +22,7 @@ june 20 2014~june twentieth twenty fourteen
 1975~nineteen seventy five
 1155~eleven fifty five
 2006-08-05~august fifth two thousand six
-august 23, 2002~august twenty third two thousand two
+august 23, 2002~august twenty third ,two thousand two
```

Comment:
is it intended to have space before comma but no space after in the verbalization?

Reply:
I would prefer `august twenty third, two thousand two` if it is possible.

Reply:
changed to `august 23, 2002~august twenty third , two thousand two` (default nemo format for punctuation marks). @Oktai15 `--punct_post_process` flag would fix the spaces